sympy/physics/units/definitions.py

```diff
@@ -1,9 +1,12 @@
-from sympy import pi, Rational, sqrt
+from sympy import Rational, pi, sqrt
 from sympy.physics.units import Quantity
-from sympy.physics.units.dimensions import length, mass, force, energy, power, pressure, frequency, time, velocity, \
-    impedance, voltage, conductance, capacitance, inductance, charge, magnetic_density, magnetic_flux, current, action, \
-    amount_of_substance, luminous_intensity, temperature, acceleration
-from sympy.physics.units.prefixes import kilo, milli, micro, nano, pico, deci, centi
+from sympy.physics.units.dimensions import (
```

Comment:
You can choose between different line output modes in https://github.com/timothycrosley/isort#multi-line-output-modes

I guess you want something between 0 and 4 like:

```python
from foo.bar.baz import (abc,
    def, ghi, klm)
```
but then the next line should start with opening bracket.

Other alternative is "hanging indent", but somebody told me that the open bracket style is used more widely in SymPy.

```python
from third_party import \
    lib1, lib2, lib3, \
    lib4, lib5, lib6
```